### DISCUSSION TITLE
Unable to use private repository

### DISCUSSION THREAD

**Discussion by leonshaw** [community/community#22261] [New to GitHub]
<p>I am able to create private repository now (free account). But when fetching/pushing, I got error:</p>
<p>remote: Your account is disabled. Please see <a href="https://github.com/account/billing" rel="nofollow noopener">https://github.com/account/billing</a></p>
<p>One thing I remember that might be relevant is that I joined student account plan which has been expired years ago.</p>
<p>Anyone know how to fix this?</p>

**Comment by c0b41** [comment]
<p>I have same problem :confused:</p>

**Comment by brunnopleffken** [comment]
<p>Yeah, I’m getting this exact same issue! I created a new private repo using the web interface but when running <em>git push --all origin</em> I get this error: “Your account is disabled.”</p>
<p>Worth to mention that I was a paid subscriber until two or three months ago, downgraded to free plan since then.</p>

**Comment by mpboom** [comment]
<p>I think this might have been due to the big changes made yesterday to the subscription system. Is the problem still there for all of you?</p>

**Comment by patpadgett** [comment]
<p>I am having the same problem.  Trying to clone a new private repo returns error saying that my account is disabled.</p>

**Comment by mpboom** [comment]
<p>I’d suggest to wait until tomorrow to allow the system to process the change, and then contact GitHub Support directly.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>If you are still experiencing this issue please contact us at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> and we’ll take a look. Thanks for your patience!</p>

**Comment by leonshaw** [comment]
<p>Thanks. I sent a support request.</p>

**Comment by toddjonker** [comment]
<p>What was the outcome?  I’m having this situation as well, with a phantom “billing issue” disabling a free private repo.</p>

**Comment by mpboom** [comment]
<p>Hi <a class="mention" href="/toddjonker">@toddjonker</a> ,</p>
<p>Here on the Community Forum we are rarely able to help you out with billing issues due to the fact that billing is something specific to your account. Therefore I think the quickest way to get help is to contact GitHub support directly at <a href="https://git.io/contact/" rel="nofollow noopener">https://github.com/contact/</a> .</p>

**Comment by toddjonker** [comment]
<p>Thanks, I’ve already done that separately and awaiting a response.  I asked because this doesn’t actually seem to have anything to do with billing.</p>
